cli/src/athenapdf.js

```diff
@@ -118,8 +123,15 @@ if (process.platform === "linux") {
     };
 }
 
+// Add custom headers if specified
+var extraHeaders = athena.httpHeader;
+
+// Toggle cache headers
+if (athena.cache) {
```

Comment:
This should be `if (!athena.cache) {`. That is, we want to set the `no-cache` header if `--no-cache` is passed in. It is why we returned a blank string before.